redfish/eventdestination.go

```diff
@@ -377,8 +382,8 @@ func CreateEventDestination(
 	if oem != nil {
 		s.Oem = oem
 	}
-
```

Comment:
Leave line spacing as it was here and on line 386.

Reply:
done

Reply:
Missed commit?